plexlibrary/logs.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
```

Comment:
I don't really see the point of this file if we're just wrapping logging methods anyway?

Reply:
Sorry, my bad, that's just my own fast-and-dirty way of handling logging across multiple files with only one logging instance. Do you want me to remove it?

Reply:
We can keep it for now.